pkg/autoscaler/config/config.go

```diff
@@ -67,6 +67,14 @@ type Config struct {
 	// the number of activators per revision.
 	ActivatorCapacity float64
 
+	// AllowZeroInitialScale indicates whether DefaultInitialScale and
+	// autoscaling.internal.knative.dev/initialScale are allowed to be set to 0.
+	AllowZeroInitialScale bool
+
+	// DefaultInitialScale is the cluster-wide initial revision size for newly deployed
```

Comment:
toooootal nit.

```suggestion
	// DefaultInitialScale is the cluster-wide default initial revision size for newly deployed
```

Reply:
Having said this, thinking out loud, I wonder if `DefaultX` is actually a bit inconsistent with the other fields here which are almost all overridable on the revision too but do not start with `Default`, either here or in their config-map property names? e.g.  `panic-threshold-percentage` isn't called `default-panic-threshold-percentage` and `pod-autoscaler-class` isn't called `default-pod-autoscaler-class` (and similarly pretty much all the other fields).

Is there a good reason to explicitly have this field name start with `Default` (and config-map key start with `default-`) here vs. staying consistent with the various other also-overridable fields?

Reply:
I agree, let's drop the "Default" prefix.